src/frontend/src/Components/shared/Filters.js

```diff
@@ -150,7 +156,20 @@ export default function Filters(props) {
 
       {open &&
         <div className="filters">
+          <div className="filters-title__container">
           <h4 className="filters-title">{textOverride ? textOverride : 'Layer filters'}</h4>
+          {
+            (mapLayers && mapLayers.current.routeLayer && mapLayers.current.routeLayer.rendered) 
+            &&
+            <div>
+              <label className="filters-title__notification">
```

Comment:
This is an incorrect use of label, we can just use the div that's above for this.